README.rst

```diff
@@ -111,30 +111,29 @@ Usage
-*   `List of supported Cartridge CLI commands <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#usage>`__
-*   `Cartridge application lifecycle <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#application-lifecycle>`__
-
-*   `Creating a Cartridge application from a template <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#creating-an-application-from-a-template>`__
-*   `Building the application locally <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#building-the-application>`__
-*   `Starting the application locally <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#starting-the-application-locally>`__
-
-    -   `Configuration files <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#configuration-files>`__
-    -   `Environment variables <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#environment-variables>`__
-    -   `Overriding default options <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#overriding-default-options>`__
-
-*   `Stopping the application locally <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#stopping-the-application-locally>`__
-*   `Checking instance status <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#checking-instance-status>`__
-*   `Displaying logs <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#displaying-logs>`__
-*   `Cleaning instance files <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#cleaning-instance-files>`__
-*   `Repairing a cluster <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#repairing-a-cluster>`__
-*   `Packaging your application <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#packaging-the-application>`__
-
-    -   `Building a distribution <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#build-directory>`__
-    -   `Packing a TGZ <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#repair-commands>`__
-    -   `Packing an RPM or DEB distribution <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#rpm-and-deb>`__
-    -   `Creating a Docker image of your app <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#docker>`__
-
-*   `Configuring an installed package <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#usage-example>`__
-*   `Files to control build and packaging <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/#build-and-packaging-files>`__
+*   `Enable shell completion for Cartridge CLI <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/installation/#enable-shell-completion>`__
+*   `List of supported Cartridge CLI commands <https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/commands/>`__
```

Comment:
Well, since there are some renaming activity in README, why not make links consistent with pages they lead to? Like `Supported Cartridge CLI commands` instead of `List of supported Cartridge CLI commands`

Reply:
It is also applicable to other links

Reply:
I'm not doing it because the titles of those pages are not consistent. Some of them are gerunds, and some of them are infinitives. Contrariwise, it's nice when a list of links in README looks consistent grammatically.
[In the doc](https://www.tarantool.io/en/doc/latest/book/cartridge/cartridge_cli/), link text is not consistent with titles either. I think it looks good enough.